samples/dotnetsay/dotnetsay.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
```

Comment:
The sample README says 2.1 or higher (twice) - maybe it is ok to leave it at 2.1? Shouldn't it build just fine even if only 2.2 is installed?

Reply:
I guess 2.0 is just OK.